packages/application-shell/src/components/navbar/navbar.mod.css

```diff
@@ -59,9 +59,9 @@
   position: relative;
   display: block;
 }
-.item__active .item-link {
+/* .item__active .item-link {
   padding-right: var(--spacing-l);
-}
+} */
```

Comment:
should we remove this commented code?

Reply:
Forgot to remove them, thanks! https://github.com/commercetools/merchant-center-application-kit/pull/2945/commits/a66a219f731a1ac7eb7229de7e41c256a9c69823